cypress/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "types": ["node", "cypress", "@testing-library/cypress"]
   },
-  "include": ["cypress/**/*", "../cypress.config.ts", "../node_modules/cypress"],
+  "include": ["cypress/**/*", "../cypress.config.ts", "../node_modules/cypress", "**/*.ts"],
```

Comment:
Allows cypress scope to be limited to cypress files so the cypress test's expects, describes etc don't overwrite jests (now we don't have to explicitly import jests globals)